app/components/pages/components/index.js

```diff
@@ -1,4 +1,9 @@
-export const Components = () =>
-  <h1>Components</h1>
+import {dispatch} from '/store'
+import {openModal} from '/components/elements/modal/actions'
+import BaseComponents from './base'
 
-export default Components
+export default () => {
```

Comment:
Also anal, any reason not to keep this as JSX?

Reply:
ugh, yeah I probably should switch it to JSX. I prefer javascript syntax in index.js beccause of convenience of `Component({variable, name, equals, prop, name})` instead of `<Component variable={variable} ... />`

Reply:
That's fair.

Reply:
yeah but then there's that bug if component is a class component with like __r and weird error message.